src/core/imap/MCIMAPSession.cpp

```diff
@@ -355,6 +355,7 @@ void IMAPSession::init()
     mAutomaticConfigurationDone = false;
     mShouldDisconnect = false;
     mLoginResponse = NULL;
+    mLoginUnparsedResponseData = NULL;
```

Comment:
Could you change the name to mUnparsedResponseData since we might also apply to other areas than just login?

FETCH response can fail to be parsed too on some broken servers :(